bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -19,254 +19,131 @@
-            {% endif %}
-          </p>
-        </li>
-        <li id="feature-account">
-          <h3 class="heading-tertiary">{{ _('Do more with a Firefox Account') }}</h3>
-
-          <p>
-            {{ _('To start a conversation with someone directly, simply create a free Firefox Account or sign in if you have one already &mdash; it’s the same account you use with Sync.') }}
-            {{ _('You can then add contacts to keep them close at hand.') }}
-          </p>
+        <p class="tagline fx-up-to-date">{{ _('Instantly browse any Web page with a friend.') }}</p>
+        <a class="try-hello-button" href="https://support.mozilla.org/kb/where-firefox-hello-button">{{ _('Try Firefox Hello') }}</a>
+        <img src="{{ static('img/firefox/hello/firefox-hello-logo-white.svg') }}" class="hello-wordmark" alt="Firefox Hello" width="380" data-fallback="true" data-png="{{ static('img/firefox/hello/firefox-hello-logo-white.png') }}">
+      </header>
+    </div>
+  </div>
+  <aside id="conditional-download-bar">
+    <div class="container">
+      <h2 class="fx-out-of-date">{{ _('Get the latest version of Firefox to start browsing Web pages with your friends.') }}</h2>
+      <h2 class="non-fx">{{ _('Get Firefox to start browsing Web pages with your friends.') }}</h2>
+      {{ download_firefox(small=True, icon=False, platform='desktop', force_direct=true, dom_id='download-fx-primary') }}
+      <div class="mobile">
+        <h2>{{ _('Use Firefox on your desktop or laptop to start browsing the Web with your friends.') }} <a class="more" href="{{ url('firefox.desktop.index') }}">{{ _('Learn more')}}</a></h2>
+        <a href="{{ url('firefox.desktop.index') }}">
+          <img src="{{ static('img/firefox/hello/desktop-hello.svg') }}" alt="" width="205">
+        </a>
+      </div>
+    </div>
+  </aside>
+  <section id="use-cases">
+    <div class="container">
+      <h2>{{ _('Get on the same page and make faster decisions in real time.') }}</h2>
+      <ul>
+        <li class="plan">
+          <h3>{{ _('Plan a trip') }}</h3>
+          <p>{{ _('Figure out the details when planning a getaway with someone else.') }}</p>
         </li>
-        <li id="feature-getfx">
-          <h3 class="heading-tertiary">{{ _('Get Firefox to get started') }}</h3>
-
-          <p>
-            {{ _('Want to start using Hello?') }}
-            {{ _('All you need is Firefox &mdash; the open source browser that helps you stay in control of your online life while you stay connected with friends and family.') }}
-            {{ _('Download Firefox to start your first conversation.') }}
-          </p>
+        <li class="shop">
+          <h3>{{ _('Shop online') }}</h3>
+          <p>{{ _('Share and compare ideas when buying a gift for a mutual friend.') }}</p>
         </li>
       </ul>
-    </div>{# /.container #}
+    </div>
   </section>
-
-  <section id="download-try">
+  <div id="features">
+    <div class="split-container quick">
+      <section>
```

Comment:
I guess I would do it be writing specific CSS rules for each of the containing elements within the current `<section>` elements, but I think this may be more verbose than just letting a containing element handling things (especially when it comes to RTL). Happy to rewrite if you think this is worth it though.